q2_fmt/_engraftment.py

```diff
@@ -52,27 +57,34 @@ def engraftment(
 
 
 def group_timepoints(
-        diversity_measure: pd.Series, metadata: qiime2.Metadata,
+       diversity_measure: pd.Series, metadata: qiime2.Metadata,
         distance_to: str, time_column: str, reference_column: str = None,
+        group_column: str = False,
         subject_column: str = False, control_column: str = None,
         filter_missing_references: bool = False,
         baseline_timepoint: str = None,
         where: str = None) -> (pd.DataFrame, pd.DataFrame):
 
+    print("baseline_timepoint")
```

Comment:
lil stray print statements (lines 68-69) 🖨️

Reply:
Thanks for the catch!